src/sections/Projects/Sistent/sistent.style.js

```diff
@@ -645,13 +645,15 @@ const SistentWrapper = styled.div`
 
   .product_cards .card {
     width: 400px;
+    height: 300px;
```

Comment:
will you make it little less like 280 px

Reply:
Okay sir sure